src/cascadia/TerminalApp/TerminalApp.vcxproj

```diff
@@ -18,85 +19,28 @@
   </PropertyGroup>
   <!-- ========================= XAML files ======================== -->
   <ItemGroup>
-    <!-- HERE BE DRAGONS:
-      For any types that use XAML information, if their .idl, .cpp, and .h
-      aren't all marked DependentUpon the corresponding .xaml file,
-      XamlTypeInfo.g.h won't pick it up correctly.
-    -->
-    <ApplicationDefinition Include="App.xaml">
-      <SubType>Designer</SubType>
-    </ApplicationDefinition>
+    <!-- DON'T PUT XAML FILES HERE! Put them in TerminalAppLib.vcxproj -->

```

Comment:
Just FYI, people doing things in Visual Studio with the click/drag aren't going to see this... but hopefully we'll catch it on PR.

Reply:
Yea I got chu. Hopefully this is enough of a warning for us to be able to see it. Even if they do add it to that project in VS, it's likely to fail spectacularly, so hopefully people will catch it before pushing.